avocado/formatters.py

```diff
@@ -287,7 +295,7 @@ def get_meta(self, exporter=None):
 
         return meta
 
-    def to_string(self, value, **context):
+    def to_string(self, value, field, context):
```

Comment:
It seems that some of these `to*` methods use the `field` so I guess it's OK to keep that in there to keep the method definition the same. However, I don't see a case of the `context` being used. Can the `to_*` definition be updated to `to_*(self, value, field):` to remove the unused context arg?